### PR TITLE
Issues #68 & #116 - Fix issue with the schedule on mobile

### PR DESCRIPTION
**Pull request by sergesemashko**
@mai @willykaram please review

### CODE REVIEW

File: sites/all/themes/zen_avenue/css/avenueresponsive.css [hunk 1]
```diff
@@ -106,6 +106,8 @@
 	}
 
     .region-sidebar-first {
+        position: absolute;
+        top: 0;
         left: -100%;
         width: auto;
         height: 3000px;
```
Comment (likewhoa) [inline sites/all/themes/zen_avenue/css/avenueresponsive.css]: Why do we need such a high height? This is what I believe is causing the big gap when viewing on some mobile devices.

File: sites/all/themes/zen_avenue/css/avenueresponsive.css [hunk 2]
```diff
@@ -106,6 +106,8 @@
 	}
 
     .region-sidebar-first {
+        position: absolute;
```
Comment (likewhoa) [inline sites/all/themes/zen_avenue/css/avenueresponsive.css:111]: Not so happy about the use of absolute positioning or the crazy height.

File: sites/all/themes/zen_avenue/css/avenueresponsive.css (hunk 1 above)
Comment (sergesemashko) [inline sites/all/themes/zen_avenue/css/avenueresponsive.css]: Actually, I'm not fan of this solution, I just didn't want to change the structure make something different from already created menu. I replaced it by the trick with streching block till the end of the page.

File: sites/all/themes/zen_avenue/css/avenueresponsive.css (hunk 2 above)
Comment (sergesemashko) [inline sites/all/themes/zen_avenue/css/avenueresponsive.css:111]: Actually, I'm not fan of this solution, I just didn't want to change the structure make something different from already created menu. I replaced it by the trick with stretching block till the end of the page.

**Comment by sergesemashko** [comment]
@mai @likewhoa please review

File: sites/all/themes/zen_avenue/css/avenueresponsive.css [hunk 3]
```diff
@@ -107,6 +107,22 @@
 		margin-bottom: 0;
 	}
 
+    .region-sidebar-first {
```
Comment (likewhoa) [inline sites/all/themes/zen_avenue/css/avenueresponsive.css:110]: This was removed by me as we don't need all this in there.

File: sites/all/themes/zen_avenue/css/avenue.css [hunk 4]
```diff
@@ -734,7 +734,9 @@ a {
 
 .sideslide #mobile-nav {
 	left: 0;
-	height: 1000px;
+    /* Trick with streching height to the end of page*/
```
Comment (likewhoa) [inline sites/all/themes/zen_avenue/css/avenue.css:737]: We don't need this crazy _trick_, we can actually get rid of the height value and perhaps set width: 100% and have an icon to close the mobile menu.

**Comment by likewhoa** [comment]
talked to @Sergey-Semashko and we decided to close this for now, he plans on making further adjustments later.

**Pull request closed without merging**